### ISSUE TITLE
Skipped rows throw KeyError

### ISSUE THREAD

**Issue by Gunner245** [django-import-export/django-import-export#1958]
**Describe the bug**
Key error is thrown if rows are skipped during import:

Exception Type:  KeyError
Exception Value: 'skip'

/.venv/lib/python3.12/site-packages/import_export/admin.py, line 573, in _create_log_entries 

**To Reproduce**
1. Setup a resource where `skip_unchaned = True`
2. Run an import file where some rows would be skipped.

**Versions (please complete the following information):**
 - Django Import Export: 4.1.1
 - Python 3.12.5
 - Django 5.1.1

**Expected behavior**
No Error to be thrown

**Problem Code**
```
    def _create_log_entries(self, user_pk, rows):
        logentry_map = {
            RowResult.IMPORT_TYPE_NEW: ADDITION,
            RowResult.IMPORT_TYPE_UPDATE: CHANGE,
            RowResult.IMPORT_TYPE_DELETE: DELETION,
        }
        for import_type, instances in rows.items():
            action_flag = logentry_map[import_type]
            self._create_log_entry(user_pk, rows[import_type], import_type, action_flag)
```

It seems its because logentry_map doesn't contain one for skip.

**Workaround**

Override this function to add handling:

```
class ImportModelAdmin(ImportMixin, admin.ModelAdmin):
    
    def _create_log_entries(self, user_pk, rows):
        logentry_map = {
            RowResult.IMPORT_TYPE_NEW: ADDITION,
            RowResult.IMPORT_TYPE_UPDATE: CHANGE,
            RowResult.IMPORT_TYPE_DELETE: DELETION,
        }
        for import_type, instances in rows.items():
            if not import_type == RowResult.IMPORT_TYPE_SKIP:
                action_flag = logentry_map[import_type]
                self._create_log_entry(user_pk, rows[import_type], import_type, action_flag)
 ```               

I'll leave it up to the maintainers to decide between adding an action flag for skip, or just not logging ones we skip.

**Comment by matthewhegarty** [comment]
Thanks for raising, this should have been fixed in #1944 and is the main branch.  Would you be able to test with the main branch and see if it fixes the issue?

**Comment by Gunner245** [comment]
I just checked and running against main fixes it for me. 

So I will use the workaround until there is a release including the fix.

```
    def _create_log_entries(self, user_pk, rows):
        logentry_map = {
            RowResult.IMPORT_TYPE_NEW: ADDITION,
            RowResult.IMPORT_TYPE_UPDATE: CHANGE,
            RowResult.IMPORT_TYPE_DELETE: DELETION,
        }
        for import_type, instances in rows.items():
            if import_type in logentry_map.keys():
                action_flag = logentry_map[import_type]
                self._create_log_entry(user_pk, rows[import_type], import_type, action_flag)
```

Thanks for the help.
